build-tools/scripts/Windows-Configuration.OperatingSystem.props

```diff
@@ -9,5 +9,8 @@
     <HostTriplet64 Condition=" '$(HostTriplet64)' == '' ">x86_64-win32</HostTriplet64>
     <HostCpuCount Condition=" '$(HostCpuCount)' == '' ">$([System.Environment]::ProcessorCount)</HostCpuCount>
     <HostBits Condition=" '$(HostBits)' == '' ">64</HostBits>
+    <JavaSdkDirectory>@JAVA_HOME@</JavaSdkDirectory>
```

Comment:
If we're going to start replacing `@...@`, please rename the file to have a `.in` extension, to make it clearer that this is a template file.